templates/supertest/put/put.handlebars

```diff
@@ -1,4 +1,4 @@
-  it('should respond with {{length description}}', function(done) {
+  it('should respond with {{length description}}{{#if inputs}}, {{inputs.body.message}}{{/if}}', function(done) {
```

Comment:
@Remco75 the description has to be truncated in the case that a large string invalidates the Javascript produced. The discussion was had towards the middle of #107 before getting sidetracked by data generation.